inbox/cs-2022.xml

```diff
@@ -0,0 +1,582 @@
+        </tr>
+        <tr>
+          <td><strong>TLS</strong></td>
+          <td align='center'>&yes;</td>
+          <td align='center'>&yes;</td>
+          <td align='center'>&yes;</td>
+          <td align='center'>&yes;</td>
+          <td>&rfc7590;</td>
+        </tr>
+        <tr>
+          <td><strong>Direct TLS</strong></td>
+          <td align='center'>&no;</td>
+          <td align='center'>&no;</td>
+          <td align='center'>&yes;<note>Server support of XEP-0368 means having the ability to accept direct TLS connections.</note></td>
+          <td align='center'>&yes;</td>
+          <td>&xep0368;</td>
+        </tr>
+        <tr>
+          <td><strong>Feature discovery</strong></td>
+          <td align='center'>&yes;</td>
+          <td align='center'>&yes;</td>
+          <td align='center'>&yes;</td>
+          <td align='center'>&yes;</td>
+          <td>&xep0030;</td>
+        </tr>
+        <tr>
+          <td><strong>Feature broadcasts</strong></td>
+          <td align='center'>&no;</td>
+          <td align='center'>&yes;</td>
+          <td align='center'>&yes;</td>
+          <td align='center'>&yes;</td>
+          <td>&xep0115;</td>
+        </tr>
+        <tr>
+          <td><strong>Server Extensibility</strong></td>
+          <td align='center'>&yes;</td>
+          <td align='center'>N/A</td>
+          <td align='center'>&yes;</td>
+          <td align='center'>N/A</td>
+          <td>&xep0114;</td>
+        </tr>
+        <tr>
+          <td><strong>Event publishing</strong></td>
+          <td align='center'>&no;</td>
+          <td align='center'>&no;</td>
+          <td align='center'>&yes;&pubsubjid;</td>
+          <td align='center'>&yes;</td>
+          <td>&xep0163;</td>
+        </tr>
+      </table>
+    </section2>
+    <section2 topic='Web Compliance Suite' anchor='web'>
+      <p>
+        To be considered XMPP web compliant, all features from the core
+        compliance category must be met, as well as all features in this suite.
+      </p>
+      <table caption='XMPP Web Compliance Levels'>
+        <tr>
+          <th>Feature</th>
+          <th>Core Server</th>
+          <th>Core Client</th>
+          <th>Advanced Server</th>
+          <th>Advanced Client</th>
+          <th>Providers</th>
+        </tr>
+        <tr>
+          <td><strong>Web Connection Mechanisms</strong></td>
+          <td align='center'>&yes;&component;</td>
+          <td align='center'>&yes;&onlyone;</td>
+          <td align='center'>&yes;&component;</td>
+          <td align='center'>&yes;&onlyone;</td>
+          <td>&rfc7395;, &xep0206; (See also: &xep0124;)</td>
+        </tr>
+        <tr>
+          <td><strong>Connection Mechanism Discovery</strong></td>
+          <td align='center'>&no;</td>
+          <td align='center'>&yes;</td>
+	  <td align='center'>N/A</td>
```

Comment:
Regarding `Connection Mechanism Discovery` - why is it `N/A` for Advanced Server but `no` for Core Server? 

See https://xmpp.org/extensions/xep-0443.html#web

Should there also be a column "deployments" were Connection Mechanism Discovery is required?